api/shape/view.js

```diff
@@ -11,40 +11,26 @@ import { ensurePages } from './Page.js';
 import { tagGeometry } from './tag.js';
 import { untagGeometry } from './untag.js';
 
-const markContent = (geometry) => {
-  if (geometry.type === 'group') {
-    return {
-      ...geometry,
-      content: geometry.content.map((child, nth) =>
-        tagGeometry(untagGeometry(child, ['groupChildId:*']), [
-          `groupChildId:${nth}`,
-        ])
-      ),
-    };
-  } else {
-    return geometry;
-  }
-};
-
-const applyModes = (options, modes) => {
+const applyModes = (shape, options, modes) => {
   if (modes.includes('wireframe')) {
-    options.wireframe = true;
+    shape = shape.tag('show:wireframe');
```

Comment:
Does this mean we can turn these on and off for individual parts of the view and not just for the whole thing?

Reply:
The view can override it, but the default is carried by the shape.